src/main/java/org/mule/extension/http/internal/request/builder/HttpResponseAttributesBuilder.java

```diff
@@ -16,15 +20,36 @@
 
   HttpResponse response;
```

Comment:
Before the commit, this attribute was used to get the others in build() method. Now it seems no longer needed